README.md

```diff
@@ -1,60 +1,77 @@
 <div align="center">
-<h1>⚛️ create-sumup-react-app ⚛️</h1>
+<h1>⚛️ @sumup/cra-template ⚛️</h1>
 
-The easiest way to get started writing a React app with SumUp's standard frontend toolchain.
+A
+[create-react-app template](https://create-react-app.dev/docs/custom-templates)
+for JavaScript React apps with SumUp's frontend stack.
```

Comment:
Do we want to provide a TypeScript template as well? (see also https://github.com/sumup/create-sumup-react-app/issues/25)

I've thought about _only_ making a TypeScript template available to enforce its adoption. It seems heavy-handed though. What do you think?

Reply:
Yes, my idea would be to work on a TypeScript template and put a big strong warning in the JS template README encouraging the use of TypeScript. The TS template would be installable by `yarn create react-app --template @sumup/typescript`.